### ISSUE TITLE
'invalid cert path' when trying to load FIDO MDS metadata items using FidoMdsMetadataItemsProvider

### ISSUE THREAD

**Issue by ChristophKaser** [webauthn4j/webauthn4j#507]
**Describe the bug**
When I try to use the FidoMdsMetadataItemsProvider to load metadata items, an exception occurs:

```
com.webauthn4j.metadata.exception.MDSException: invalid cert path
	at com.webauthn4j.metadata.FidoMdsMetadataItemsProvider.validateCertPath(FidoMdsMetadataItemsProvider.java:236)
	at com.webauthn4j.metadata.FidoMdsMetadataItemsProvider.fetchMetadataTOCPayload(FidoMdsMetadataItemsProvider.java:203)
	at com.webauthn4j.metadata.FidoMdsMetadataItemsProvider.refresh(FidoMdsMetadataItemsProvider.java:160)
	at com.webauthn4j.metadata.FidoMdsMetadataItemsProvider.provide(FidoMdsMetadataItemsProvider.java:146)
	at com.webauthn4j.metadata.MetadataItemsMetadataStatementsProvider.provide(MetadataItemsMetadataStatementsProvider.java:37)
	at com.webauthn4j.metadata.MetadataStatementsTrustAnchorsProvider.provide(MetadataStatementsTrustAnchorsProvider.java:37)
	at com.webauthn4j.anchor.TrustAnchorsResolverImpl.resolve(TrustAnchorsResolverImpl.java:57)
	at com.webauthn4j.validator.attestation.trustworthiness.certpath.TrustAnchorCertPathTrustworthinessValidator.resolveTrustAnchors(TrustAnchorCertPathTrustworthinessValidator.java:43)
	at com.webauthn4j.validator.attestation.trustworthiness.certpath.CertPathTrustworthinessValidatorBase.validate(CertPathTrustworthinessValidatorBase.java:45)
	at com.webauthn4j.validator.AttestationValidator.validate(AttestationValidator.java:121)
	at com.webauthn4j.validator.RegistrationDataValidator.validate(RegistrationDataValidator.java:158)
	at com.webauthn4j.WebAuthnRegistrationManager.validate(WebAuthnRegistrationManager.java:204)
	at com.webauthn4j.WebAuthnManager.validate(WebAuthnManager.java:161)
	at ...
Caused by: java.security.cert.CertPathValidatorException: Path does not chain with any of the trust anchors
	at java.base/sun.security.provider.certpath.PKIXCertPathValidator.validate(PKIXCertPathValidator.java:158)
	at java.base/sun.security.provider.certpath.PKIXCertPathValidator.engineValidate(PKIXCertPathValidator.java:84)
	at java.base/java.security.cert.CertPathValidator.validate(CertPathValidator.java:309)
	at com.webauthn4j.metadata.FidoMdsMetadataItemsProvider.validateCertPath(FidoMdsMetadataItemsProvider.java:232)
	... 51 more
```
I believe this is caused by a changed certificate to sign the JWS.

**To Reproduce**
```
ObjectMapper jsonMapper = new ObjectMapper();
jsonMapper.registerModule(new WebAuthnMetadataJSONModule());
ObjectMapper cborMapper = new ObjectMapper(new CBORFactory());
ObjectConverter objectConverter = new ObjectConverter(jsonMapper, cborMapper);
var provider = new FidoMdsMetadataItemsProvider(objectConverter, "MY_TOKEN");
provider.provide();
```

**Comment by ynojima** [comment]
Thank you for your report. Please download a new .cer file from FIDO Alliance and use this constructor instead:
https://github.com/webauthn4j/webauthn4j/blob/76409a88935569ed5f67147abe3878184dc6348f/webauthn4j-metadata/src/main/java/com/webauthn4j/metadata/FidoMdsMetadataItemsProvider.java#L75

**Comment by ChristophKaser** [comment]
Thank you for your response.

In case someone else is searching: The link to the cer-file in the FAQ of the FIDO Alliance is dead. The information about the changed certificate can be found here:
https://groups.google.com/a/fidoalliance.org/g/fido-dev/c/KTXB4NWAp5U (along with a deprecation notice for the MDS2 metadata service)

@ynojima Would you accept a PR that replaces the included root certificate with the new one?

**Comment by ynojima** [comment]
Thank you for suggesting PR, but now I think embedding root .cer file is not appropriate. I'll remove the constructor overload which doesn't require .cer file from next version.

**Comment by ynojima** [comment]
Fixed in #508
